include/grpc/impl/codegen/grpc_types.h

```diff
@@ -119,9 +119,13 @@ typedef struct {
     These configuration options are modelled as key-value pairs as defined
     by grpc_arg; keys are strings to allow easy backwards-compatible extension
     by arbitrary parties. All evaluation is performed at channel creation
-    time (i.e. the values in this structure need only live through the
+    time (i.e. the keys and values in this structure need only live through the
     creation invocation).
 
+    However, if one of the args has grpc_arg_type==GRPC_ARG_POINTER, then the
+    grpc_arg_pointer_vtable must live until the channel args are done being
+    used by core (i.e. when the object they were passed to is destroyed).
```

Comment:
"the object for use with which they were passed"? One doesn't pass channel args to a channel; one passes channel args to the channel creation function.

Reply:
Done